packages/editor/src/components/post-url/panel.js

```diff
@@ -44,18 +46,22 @@ export default function PostURLPanel() {
 }
 
 function PostURLToggle( { isOpen, onClick } ) {
-	const label = usePostURLLabel();
+	const slug = useSelect(
+		( select ) =>
+			safeDecodeURIComponent( select( editorStore ).getEditedPostSlug() ),
```

Comment:
Minor non blocker: In a case like this I would probably run safeDecodeURIComponent outside useSelect inside a useMemo. On every store change (e.g: character typed) we will be calling safeDecodeURIComponent even if nothing changed on the getEditedPostSlug. It is not problematic and totally fine in one useSelect call but these things add up with hundreds of useSelect calls. Ideally, I think useSelect should not do computations and return things from the store as is.